### ISSUE TITLE
issues in using library  "com.github.mancj:SlideUp-Android:$current_version"

### ISSUE THREAD

**Issue by Rupeshrajsingh** [mancj/SlideUp-Android#66]
apply plugin: 'com.android.application'

android {
    compileSdkVersion 28
    defaultConfig {
        applicationId "com.technetty.reaction"
        minSdkVersion 15
        targetSdkVersion 28
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    //noinspection GradleCompatible
    implementation 'com.android.support:appcompat-v7:28.0.0-rc02'
    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
    implementation 'com.android.support:design:28.0.0-rc01'
    implementation 'com.android.support:support-v4:28.0.0-rc02'
    testImplementation 'junit:junit:4.12'
    implementation 'pl.droidsonroids.gif:android-gif-drawable:1.2.+'
    androidTestImplementation 'com.android.support.test:runner:1.0.2'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
    implementation 'com.facebook.android:facebook-login:[4,5)'
  
<---- this line have error in "$current_version" ---->
   // implementation "com.github.mancj:SlideUp-Android:$current_version"

**Comment by Daksh14** [comment]
use implementation "com.github.mancj:SlideUp-Android:2.2.8"

**Comment by Rupeshrajsingh** [comment]
Yes, i use implementation 'com.github.mancj:SlideUp-Android:2.2.8' but still it's not work Failed to resolve: com.github.mancj:SlideUp-Android:2.2.8

**Comment by Daksh14** [comment]
Add it in the second gradle file
